test/reconciler.go

```diff
@@ -59,6 +59,14 @@ type testClientFactory struct {
 	serverURL string
 }
 
+func NewServerlessTestReconciler(client client.Client) common.Reconciler {
```

Comment:
Let's use something like NoServer or WithoutServer instead of Serverless to avoid confusion with:
https://en.wikipedia.org/wiki/Serverless_computing

Reply:
Makes sense!